paymentsheet/src/main/java/com/stripe/android/paymentsheet/injection/FlowControllerModule.kt

```diff
@@ -39,4 +41,10 @@ internal object FlowControllerModule {
     fun provideViewModelScope(viewModel: FlowControllerViewModel): CoroutineScope {
         return viewModel.viewModelScope
     }
+
+    @Provides
+    @Singleton
+    fun provideStripeImageLoader(context: Context): StripeImageLoader {
+        return StripeImageLoader(context)
+    }
```

Comment:
Nit: Could this be a `Binds` instead of a `Provides`?

Reply:
I decided not to make `StripeImageLoader` injectable since there are other params (`logger`, `memoryCache`, `networkImageDecoder`, `diskCache`). In other areas of the SDK, we might choose to not use the defaults there.